docs/workers.md

```diff
@@ -502,11 +504,32 @@ after setting this option in the shared configuration!
 This style of configuration supersedes the legacy `synapse.app.appservice`
 worker application type.
 
+#### Push Notifications
+
+You can designate one generic worker to sending push notifications to
```

Comment:
is the word "one" correct here? the text below describing `pusher_instances` suggests that we can have multiple pusher workers.